### PR TITLE
CIS: GLB Pool Development

### PR DESCRIPTION
**Pull request by KumarGanesanIBM**

### CODE REVIEW

File: ibm/resource_ibm_cis_origin_pool_test.go [hunk 1]
```diff
@@ -86,8 +91,6 @@ func TestAccIBMCisPool_FullySpecified(t *testing.T) {
 }
 
 func TestAccIBMCisPool_CreateAfterManualDestroy(t *testing.T) {
-	//t.Parallel()
-	t.Skip()
```
Comment (kavya498) [inline ibm/resource_ibm_cis_origin_pool_test.go:90]: few test cases are intentionally skipped.. can you please revert back this change..

File: ibm/resource_ibm_cis_origin_pool.go [hunk 2]
```diff
@@ -2,300 +2,383 @@ package ibm
...
+func resourceCISPoolUpdate(d *schema.ResourceData, meta interface{}) error {
+	cisClient, err := meta.(ClientSession).CisGLBPoolClientSession()
 	if err != nil {
 		return err
 	}
-	poolUpdate := v1.PoolBody{}
-	if d.HasChange("name") || d.HasChange("origins") || d.HasChange("check_regions") || d.HasChange("notification_email") || d.HasChange("monitor") || d.HasChange("enabled") || d.HasChange("minimum_origins") || d.HasChange("description") {
-		if name, ok := d.GetOk("name"); ok {
-			poolUpdate.Name = name.(string)
+	cisID := d.Get(cisID).(string)
```
Comment (kavya498) [inline ibm/resource_ibm_cis_origin_pool.go:256]: please get the cis and pool id values from d.ID

File: ibm/resource_ibm_cis_origin_pool.go [hunk 3]
```diff
@@ -2,300 +2,383 @@ package ibm
...
 						},
-						"enabled": {
+						cisGLBPoolOriginsEnabled: {
 							Type:     schema.TypeBool,
 							Required: true,
 						},
-						"weight": {
-							Type:     schema.TypeInt,
+						cisGLBPoolOriginsWeight: {
+							Type:     schema.TypeFloat,
```
Comment (kavya498) [inline ibm/resource_ibm_cis_origin_pool.go:105]: Why is it changed to float any specific reason?

File: ibm/resource_ibm_cis_origin_pool_test.go (hunk 1 above)
Comment (KumarGanesanIBM) [reply ibm/resource_ibm_cis_origin_pool_test.go:90]: done

File: ibm/resource_ibm_cis_origin_pool.go (hunk 2 above)
Comment (KumarGanesanIBM) [reply ibm/resource_ibm_cis_origin_pool.go:256]: done

File: ibm/resource_ibm_cis_origin_pool.go (hunk 3 above)
Comment (KumarGanesanIBM) [reply ibm/resource_ibm_cis_origin_pool.go:105]: weight valid values are between `0.0` to `1.0`.

**Pull request merged**